lib/zoom/actions/report.rb

```diff
@@ -40,9 +40,9 @@ def meeting_polls_report(*args)
       end
 
       def webinar_details_report(*args)
-        # TODO: implement webinar_details_report
-        # params = Zoom::Params.new(Utils.extract_options!(args))
-        raise Zoom::NotImplemented, 'webinar_details_report is not yet implemented'
+        params = Zoom::Params.new(Utils.extract_options!(args))
+        params.require(:id).permit(%i[page_size next_page_token])
```

Comment:
page_size & next_page_token are redundant params

Reply:
I don't believe they are. But I also can't find any place in the docs where these keys are referenced.

Am I looking in the right place? https://marketplace.zoom.us/docs/api-reference/zoom-api/reports/reportwebinardetails

Reply:
Yes. For this endpoint, the only required parameter is id

Reply:
For example, for endpoint [/report/webinars/{webinarId}/participants](https://marketplace.zoom.us/docs/api-reference/zoom-api/reports/reportwebinarparticipants), these parameters are allowed

Reply:
I see. Yep you're totally right. Should just be
```suggestion
        params.require(:webinar_id)
```